### PR TITLE
[share_plus] pubspec file updated

### PR DESCRIPTION
**Pull request by pratikbutani**
## Description

- Updated the latest dependencies in pubspec.yaml

## Related Issues

- Update the latest version of all dependcies

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated the version in `pubspec.yaml` and `CHANGELOG.md`.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I am willing to follow up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/fluttercommunity/plus_plugins/blob/main/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

File: packages/share_plus/share_plus/example/lib/main.dart [hunk 1]
```diff
@@ -30,7 +30,7 @@ class DemoAppState extends State<DemoApp> {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Share Plus Plugin Demo',
+      title: 'Share Plus Plus Plugin Demo',
```
Comment (miquelbeltran) [inline packages/share_plus/share_plus/example/lib/main.dart:33]: why adding Plus twice?

**Comment by miquelbeltran** [comment]
Thanks, but this is a low quality contribution and won't qualify for Hacktoberfest

File: packages/share_plus/share_plus/example/lib/main.dart (hunk 1 above)
Comment (pratikbutani) [reply packages/share_plus/share_plus/example/lib/main.dart:33]: Sorry for that, let me update it.

**Comment by pratikbutani** [comment]
> Thanks, but this is a low quality contribution and won't qualify for Hacktoberfest

Yeah still I am updating the code, will give you the updated PR soon.

**Comment by vbuberen** [comment]
> Thanks, but this is a low quality contribution and won't qualify for Hacktoberfest

I think it also doesn't bring any value as it doesn't update any outdated dependencies, which block usage of plugins for some users, as there are no such dependencies at the moment.

> Yeah still I am updating the code, will give you the updated PR soon.

If you are going to just change the title in the example as you commented after review - don't do it. If you have other ideas that might actually be valuable (for example, pick one of opened issues we have) - feel free to open a new PR as I am going to close this one.

**Pull request closed without merging**